bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="haywardomnilogic" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Hayward OmniLogix Binding</name>
+	<description>Binding for the Hayward OmniLogix swimming pool automation controller.</description>
+	<author>Matt Myers</author>
```

Comment:
The author tag is deprecated and should therefore be removed. See openhab/openhab-core#1844.